README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io.svg?branch=develop)](https://travis-ci.org/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io)
```

Comment:
@Logan1x Take a look at this https://github.com/nasa/podaacpy. The badges are usually below the Title of the project and above the logo (if there is one).